### PR TITLE
[date](parser) Support DateV1 keyword

### PR DESCRIPTION
**Pull request by Gabriel39**
## Proposed changes

image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Gabriel39** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4 [hunk 1]
```diff
@@ -873,6 +875,8 @@ nonReserved
     | DATETIME
     | DATETIMEV2
     | DATEV2
+    | DATETIMEV1
+    | DATEV1
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:879]: nereids should update LogicalPlanBuilder and other files too.
please add create table and query test for legacy planner and Nereids

**Comment by yiguolei** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by mrhhsg — APPROVED** [review]
lgtm

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**